industry-themes/fx_client_services/ksqlDB-cloud-setup.sh

```diff
@@ -0,0 +1,49 @@
+#create kafka cluster in UI
+
+#creata a kafka API key/secret in UI
+
+#get bootstrap server
+pkc-ep9mm.us-east-2.aws.confluent.cloud:9092
```

Comment:
should this be masked? It's not of use on its own, but it is confusing

Reply:
True, just added a commit with more masking.